classroom.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
+  attr_accesor :label
+  attr_reader :students
+
+  def initialize(_label)
+    @label = lebel
+    @classroom = []
```

Comment:
- [ ] I believe you meant to initialize `@students` array rather than `@classroom` in order to implement the `has-many` / `belongs-to` relationships between `Clasroom` and `Student`

Reply:
no the student is inside the array classroom